packages/gatsby-plugin-gatsby-cloud/.gitignore

```diff
@@ -1,7 +1,10 @@
 yarn.lock
-/*.js
+/**/*.js
```

Comment:
Also seems unrelated

Reply:
Yeah we added that to make sure it captures the right js files

Reply:
If we remove the TypeScript `models` files the file can stay the same